app/components/avo/field_wrapper_component.rb

```diff
@@ -99,14 +78,7 @@ def data
   end
 
   def stacked?
-    # Override on the declaration level
-    return @stacked unless @stacked.nil?
-
-    # Fetch it from the field
-    return field.stacked unless field.stacked.nil?
-
-    # Fallback to defaults
-    Avo.configuration.field_wrapper_layout == :stacked
+    @stacked || field.stacked || Avo.configuration.field_wrapper_layout == :stacked
```

Comment:
`nil` and `false` are both falsey, so I think we can just make this a simple OR fallback.